P0009/P0009.md

```diff
@@ -1607,97 +1613,112 @@ typename M::size_type
 typename M::layout_type
 ```
 [4]{.pnum} *Result:* A type `MP` which meets the layout mapping policy requirements ([mdspan.layoutpolicy.reqmts]) and for which 
-                     `is_same_v<typename MP::template mapping<typename M::extents_type>, M>` is `true`.
+                     `typename MP::template mapping<typename M::extents_type>` denotes the type `M`.
 
 ```c++
 m.extents()
 ```
 [5]{.pnum} *Result:* `const typename M::extents_type&`
 
-[6]{.pnum} *Returns:* `e`
-
 ```c++
 m(i...)
 ```
-[7]{.pnum} *Result:* `typename M::size_type`
+[6]{.pnum} *Result:* `typename M::size_type`
+
+[7]{.pnum} *Returns:* A value less then `numeric_limits<typename M::size_type>::max - 1`.
+
+```c++
+m(i...) == m(static_cast<typename M::size_type>(i)...)
+```
+
+[8]{.pnum} *Result:* `bool`
 
-[8]{.pnum} *Postcondition:* `m(i...)==m(static_cast<typename M::size_type>(i)...)` is `true`.
+[9]{.pnum} *Value:* `true`
 
 ```c++
 m.required_span_size()
 ```
-[9]{.pnum} *Result:* `typename M::size_type`
+[10]{.pnum} *Result:* `typename M::size_type`
 
-[10]{.pnum} *Returns:* If the multidimensional index space that `e` defines has size $0$, then `0`,
-      else `1` plus the maximum value of `m(i...)` for all `i...` in `e`.
+[11]{.pnum} *Returns:* If the multidimensional index space `m.extents()` has size $0$, then `0`,
+      else `1` plus the maximum value of `m(i...)` for all `i`.
```

Comment:
LWG really wanted us not to specify "for all `i` in the domain"?  This sounds uncomfortably like "for all integer packs `i`."

Reply:
but i is defined to be in the domain